src/Illuminate/Cookie/Middleware/EncryptCookies.php

```diff
@@ -214,7 +216,15 @@ protected function duplicate(Cookie $cookie, $value)
      */
     public function isDisabled($name)
     {
-        return in_array($name, array_merge($this->except, static::$neverEncrypt));
+        $exclude = Collection::make($this->except)
+            ->merge(static::$neverEncrypt)
+            ->filter(fn ($cookie) => Str::contains($cookie,'*'));
+
+        return $exclude->isEmpty()
+            ? in_array($name, array_merge($this->except, static::$neverEncrypt))
+            : $exclude->filter(fn ($cookie) => Str::startsWith(
+                $name, Str::replace('*', '', $cookie)
+            ))->isNotEmpty();
```

Comment:
Could be simplified by using `some()` over chaining `filter()` with `isEmpty()`
```suggestion
        $exclude = Collection::make($this->except)
            ->merge(static::$neverEncrypt)
            ->filter(fn ($cookie) => Str::contains($cookie,'*'));

        return $exclude->isEmpty()
            ? in_array($name, array_merge($this->except, static::$neverEncrypt))
            : $exclude->some(fn ($cookie) => Str::startsWith(
                $name, Str::replace('*', '', $cookie)
            ));
```

Reply:
Also, seems like it would be easier if the class property `$except` would be a collection (especially, with the comment below) 🤔

Reply:
Not sure if you still want to simplify the `filter(...)->isNotEmpty()` to `some(...)`